helm/fluent-logshipping-app/values.yaml

```diff
@@ -25,7 +25,7 @@ fluentbit:
   image:
     name: giantswarm/fluent-bit
     # -- Overrides the image tag whose default is the chart's appVersion
-    tag: ""
+    tag: "1.7.1-systemd247"
```

Comment:
Maybe there's an upstream version with this fix? ... maybe

Reply:
Sadly no this is in 1.9 and 1.8 but not in 1.7